src/opm/parser/eclipse/EclipseState/Grid/FieldProps.cpp

```diff
@@ -705,24 +725,24 @@ void FieldProps::handle_int_keyword(const DeckKeyword& keyword, const Box& box)
 }
 
 
-void FieldProps::handle_double_keyword(const DeckKeyword& keyword, const Box& box) {
+void FieldProps::handle_double_keyword(Section section, const DeckKeyword& keyword, const Box& box) {
     auto& field_data = this->init_get<double>(keyword.name());
     const auto& deck_data = keyword.getSIDoubleData();
     const auto& deck_status = keyword.getValueStatus();
-    assign_deck(keyword, field_data, deck_data, deck_status, box);
-}
 
+    if (section == Section::EDIT && keywords::multiplier_keywords.count(keyword.name()) == 1)
+        multiply_deck(keyword, field_data, deck_data, deck_status, box);
+    else
+        assign_deck(keyword, field_data, deck_data, deck_status, box);
```

Comment:
I haven't thought this through so I'll just ask.  Does this criterion cover all cases?  This will `assign_deck` if we're **not** in `EDIT` or if the keyword is not one of the multiplier ones.  Is that the appropriate course of action?

Reply:
> This will assign_deck if we're not in EDIT or if the keyword is not one of the multiplier ones. Is that the appropriate course of action?

It is the intended behavior and what is currently implemented in the `Eclipse3DProperties` implementation, but the behavior of keywords in the `GRID` versus the `EDIT` section is less than clear (see: https://github.com/OPM/opm-simulators/issues/2183) - and I am not willing to swear that current implementation is Right&trade; - that being said a point of the `FieldProps` implementation has been that it should be easier to implement section aware behavior - *when* we really understand what is the appropriate course of action.

Reply:
There is also the nagging issue in the back of my mind that at least the `MULT*` keywords **could** be present in the `SCHEDULE` section too.  I don't know if we really support that at the moment.

Reply:
That is right - we have some *extremely* basic infrastructure in place for this, but that goes to the Schedule class.